FAQs.md

```diff
@@ -0,0 +1,111 @@
+**What is Images-to-pdf?**
+
+Images-to-pdf is an Android utility software that can convert your image files to pdf in few seconds which is the basic and primary feature of the software. The software has a lot of other useful features and they include;
+- Create PDF from multiple images from camera or gallery
+- View your converted PDFs
+- Open, Rename, Delete, print, share files
+- Sort the files order based on a number of options
+- View File details (Path, size, date created ...)
```

Comment:
Add QR code scanning code option also...

`Convert QR code or Bar code scan results to PDF`